sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -47,322 +44,259 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-ai-formrecognizer</artifactId>
-        <version>3.0.8</version>
+        <version>3.1.1</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-ai-textanalytics</artifactId>
-        <version>5.0.6</version>
+        <version>5.0.7</version>
       </dependency>
-
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-communication-chat</artifactId>
-        <version>1.0.1</version>
-      </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-common</artifactId>
-        <version>1.0.1</version>
+        <version>1.0.2</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-identity</artifactId>
-        <version>1.1.0</version>
+        <version>1.1.1</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-phonenumbers</artifactId>
-        <version>1.0.2</version>
+        <version>1.0.3</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-sms</artifactId>
-        <version>1.0.2</version>
+        <version>1.0.3</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core</artifactId>
-        <version>1.16.0</version>
+        <version>1.17.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-amqp</artifactId>
-        <version>2.0.5</version>
+        <version>2.2.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-http-netty</artifactId>
-        <version>1.9.2</version>
+        <version>1.10.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-http-okhttp</artifactId>
-        <version>1.6.2</version>
+        <version>1.7.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-management</artifactId>
-        <version>1.2.2</version>
+        <version>1.3.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-gson</artifactId>
-        <version>1.1.2</version>
+        <version>1.1.3</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-jackson</artifactId>
-        <version>1.2.3</version>
-      </dependency>
-
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-core-test</artifactId>
```

Comment:
Why do we include this library in our BOM?

Reply:
This should be removed

Reply:
Why do we want to remove this? I think spring team may need this?

Reply:
Can you please share a scenario here?

Reply:
Spring libraries should start taking a dependency on released BOMs. They will need some test infrastructure provided by `azure-core-test` for their own testing.

Reply:
Also, what's the downside to including `azure-core-test` in BOM?

Reply:
Even if that were the case - would azure-core-test still not be a dependency for spring? So, should it not be in the spring BOM or some other BOM? azure-sdk-bom should have SDKs and external closures for them. Test dependency in it seems odd.

Reply:
Spring BOM will include azure-sdk-bom just like we include netty and reactor boms. They should not have to include azure-sdk-bom and azure-core-test dependency separately. Also, previous versions of `azure-sdk-bom` includes `azure-core-test`. So, removing it is also a breaking change - https://mvnrepository.com/artifact/com.azure/azure-sdk-bom/1.0.3